### PR TITLE
[PT-BR] Add `Skinning`

### PR DESCRIPTION
**Pull request by Icaro1447**
gonna do skin.ini next

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Skinning/pt-br.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Skinning
+
+Skinning é um dos principais recursos do osu!. Possibilita os jogadores derivarem os elementos originais de skins para criar a sua própria! Skins podem variar desde temáticas, profissionais, quase impossível de usa-las ou apenas para diversão.
```
Comment (daviirodrig) [inline wiki/Skinning/pt-br.md:3]: `derivarem` is just too formal, and this way would be better for understanding.
```suggestion
Skinning é um dos principais recursos do osu!. Possibilita que os jogadores utilizem os elementos originais de skins para criar a sua própria! Skins podem variar desde temáticas, profissionais, quase impossível de usá-las ou apenas para diversão.
```

File: wiki/Skinning/pt-br.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Sets de skinning
+
+*Pagina Principal: [Lista de Sets de skinning](/wiki/Ranking_Criteria/Skin_Set_List)*
```
Comment (daviirodrig) [inline wiki/Skinning/pt-br.md:7]:
```suggestion
*Página Principal: [Lista de Sets de skinning](/wiki/Ranking_Criteria/Skin_Set_List)*
```

**Review by daviirodrig — CHANGES_REQUESTED** [review]

File: wiki/Skinning/pt-br.md (hunk 1 above)
Comment (Icaro1447) [reply wiki/Skinning/pt-br.md:3]: nice thx

**Review by daviirodrig — APPROVED** [review]
Looks good to me

**Review by TicClick — APPROVED** [review]

**Pull request merged**